packages/react/src/components/ComposedModal/ComposedModal-story.js

```diff
@@ -98,12 +98,8 @@ storiesOf('ComposedModal', module)
           </p>
         </ModalBody>
         <ModalFooter>
-          <Button
-            kind={props.composedModal().danger ? 'tertiary' : 'secondary'}>
-            Cancel
-          </Button>
-          <Button
-            kind={props.composedModal().danger ? 'danger--primary' : 'primary'}>
+          <Button kind={'secondary'}>Cancel</Button>
```

Comment:
we can just do this right?

```suggestion
          <Button kind="secondary">Cancel</Button>
```